### PR TITLE
Update nav

### PR DESCRIPTION
**Pull request by Charlesdoiron**

### CODE REVIEW

File: templates/components/_footer.html.twig [hunk 1]
```diff
@@ -3,138 +3,241 @@
...
+            <a href="https://en-marche.fr/politique-protection-donnees" rel="noopener noreferrer" target="_blank">Politique de Protection des Données</a>
+          </li>
+        </ul>
+        <ul class="list__links text--gray">
+          <li>&copy; La République En Marche !</li>
+        </ul>
+      </div>
     </div>
-</footer>
+  </footer>
```
Comment (Remg) [inline templates/components/_footer.html.twig]: missing blank line at end of file

File: templates/components/_nav_mobile.html.twig [hunk 2]
```diff
@@ -149,56 +184,56 @@
...
-                    <a href="http://www.snapchat.com/add/enmarchefr" target="_blank" rel="noopener noreferrer">
+                    <a href="http://www.snapchat.com/add/enmarchefr" rel="noopener noreferrer" target="_blank">
                         <i class="fa fa-snapchat-ghost"></i>
                     </a>
                 </li>
             </ul>
         </div>
     </div>
-</nav>
+</nav>
```
Comment (Remg) [inline templates/components/_nav_mobile.html.twig]: missing blank line at end of file

File: templates/components/_footer.html.twig (hunk 1 above)
Comment (Charlesdoiron) [reply templates/components/_footer.html.twig]: 👌

File: front/style/layout/_nav.scss [hunk 3]
```diff
@@ -233,14 +272,13 @@
         padding: 15px 10px 5px;
         line-height: 24px;
         background-color: $darker;
-
+
```
Comment (ottaviano) [inline front/style/layout/_nav.scss]: CS: useless spaces

File: templates/components/_nav_mobile.html.twig [hunk 4]
```diff
@@ -93,22 +95,46 @@
...
+                </li>
+                <li>
+                    <a href="{{ path('our_referents_homepage') }}">Nos référents</a>
+                </li>
+                <li>
+                    <a href="https://dpt.en-marche.fr">Nos sites départementaux</a>
                 </li>
+                <li>
+                    <a href="{{ path('legislatives_homepage') }}">
+                        Nos députés</a>
```
Comment (ottaviano) [inline templates/components/_nav_mobile.html.twig]: CS: could you move `</a>` in the new line?

File: templates/components/_nav_desktop.html.twig [hunk 5]
```diff
@@ -1,163 +1,210 @@
...
-            <a href="{{ path('react_app_ideas_workshop_home') }}">Atelier des idées</a>
-        </li>
-        <li>
-            <a href="{{ path('page_campus') }}">Formation</a>
-        </li>
+        <div class="l__row">
+            <div class="nav-dropdown">
+                <p>
+                    Le Mouvement
+                    <img alt="menu" class="nav__chevron" src="./../../images/icons/chevron-down-light-black.svg">
```
Comment (ottaviano) [inline templates/components/_nav_desktop.html.twig]: ``

File: templates/components/_nav_desktop.html.twig [hunk 6]
```diff
@@ -1,163 +1,210 @@
...
+                        <a href="{{ path('page_le_mouvement') }}">Qui sommes-nous ?</a>
+                    </li>
+                    <li>
+                        <a href="{{ path('app_our_organization') }}">Notre organisation</a>
+                    </li>
+                    <li>
+                        <a href="{{ path('our_referents_homepage') }}">Nos référents</a>
+                    </li>
+                    <li>
+                        <a href="https://dpt.en-marche.fr">Nos sites départementaux</a>
```
Comment (ottaviano) [inline templates/components/_nav_desktop.html.twig]: `target=blanc` for this link, no?

File: templates/components/_nav_mobile.html.twig [hunk 7]
```diff
@@ -52,38 +56,36 @@
                     </ul>
                 {% endif %}
                 <ul>
-                {% if has_role_adherent and is_granted('CREATE_COMMITTEE') %}
-                    <ul>
+                    {% if has_role_adherent and is_granted('CREATE_COMMITTEE') %}
+                        <ul>
+                            <li><ahref="{{path('app_adherent_create_committee')}}">Créer un comité</a></li>
```
Comment (Remg) [inline templates/components/_nav_mobile.html.twig]: plase put spaces inside brackets.
=> `{{ path('app_adherent_create_committee') }}`

File: templates/components/_nav_desktop.html.twig [hunk 8]
```diff
@@ -1,163 +1,210 @@
...
-                                        {% endif %}
-                                        {% if app.user.isHost %}
-                                            <li class="visually-hidden">
-                                                <a href="{{ path('app_formation_intro') }}">Je me forme <span>Nouveau</span></a>
-                                            </li>
-                                        {% endif %}
-                                    </ul>
+            <p>
+                S'engager
+                <img alt="menu" class="nav__chevron" src="./../../images/icons/chevron-down-light-black.svg">
```
Comment (ottaviano) [inline templates/components/_nav_desktop.html.twig]: ``

File: templates/components/_nav_mobile.html.twig (hunk 7 above)
Comment (ottaviano) [reply templates/components/_nav_mobile.html.twig]: and here too `ahref`

Comment (Charlesdoiron) [reply templates/components/_nav_mobile.html.twig]: 👌

File: web/images/icons/chevron-down-light-black.svg [hunk 9]
```diff
@@ -0,0 +1,4 @@
+
```
Comment (ottaviano) [inline web/images/icons/chevron-down-light-black.svg]: could you remove this empty line

Comment (Charlesdoiron) [reply web/images/icons/chevron-down-light-black.svg]: ok

**Pull request merged**